flir.md

```diff
@@ -0,0 +1,43 @@
+# Infrared heat imaging data
```

Comment:
It looks like these pages are per-sensor, and would describe the data sets at many stages of processing (level 0, 1, etc). If so, they can be named after the sensors